### ISSUE TITLE
accessing value property of multiprocessing.Value yields warning

### ISSUE THREAD

**Issue by nresare** [python/typeshed#1479]
Instances of the `multiprocesising.Value` type has a `value` property that can be used for reading and writing. This is quite clear from [the API docs](https://docs.python.org/3/library/multiprocessing.html#multiprocessing.Value) that this is the way to go, yet mypy yields `error: "Value" has no attribute "value"` when attempting to access `value` of a `Value` instance.

**Comment by nresare** [comment]
A trivial example illustrating the problem:
```
from multiprocessing import Value
v = Value('i', 42)
print("value is {}".format(v.value))
```
Running `mypy mypy_value.py` yields `error: "Value" has no attribute "value"` but executes fine as would be expected in python 3.6

**Comment by JelleZijlstra** [comment]
Fixed by #1480.
